README.md

```diff
@@ -117,12 +117,17 @@ To load statutes into elasticsearch, run:
 python manage.py index_statutes
 ```
 
+#### Connecting to a RDS DB instance instead of local DB
+
 *Note: FEC and 18F members can set the SQL connection to one of the RDS boxes with:*
 
 ```
 export SQLA_CONN=<psql:address-to-box>
 ```
 
+Warning: never perform 'update all' when pointing to an RDS box via the SQLA_CON
+N env var
```

Comment:
It looks like either an errant press of the enter key made it in or the line wrap settings in your editor are causing things to actually kick down to the next line with a line break, which also causes extra whitespace to show up, e.g.,:

<img width="941" alt="screen shot 2017-12-05 at 9 34 43 am" src="https://user-images.githubusercontent.com/121565/33612569-fb7a7bec-d99f-11e7-9296-08aef755b0f4.png">

Please double check or just remove the extra space so we don't confuse folks who are trying to set things up and set this environment variable. :-)